# This message represents a full human pose 2D as a list of keypoints

Header header

# The id of the pose
int32 pose_id

# The pose detection confidence of the model
float32 conf

# A list of a human 2D pose keypoints
Pose2DKeypoint[] keypoint_list
